vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -343,8 +456,8 @@ def get_rope(
         return _ROPE_DICT[key]
 
     if rope_scaling is None:
-        rotary_emb = RotaryEmbedding(head_size, rotary_dim, max_position, base,
-                                     is_neox_style)
+        rotary_emb = LlamaRotaryEmbedding(head_size, rotary_dim, max_position, base,
+                                          is_neox_style)
```

Comment:
Check if prior RotaryEmbedding class is still needed, and if not, replace it fully with the HPU-enabled one.